### PR TITLE
Add option to build without libertine

### PR DESCRIPTION
**Pull request by mariogrip**

### CODE REVIEW

**Comment by Flohack74** [comment]
Is this a preparation to fix content-hub for the Pinephone (and all other Non-Android devices) ?

**Comment by dobey** [comment]
> Is this a preparation to fix content-hub for the Pinephone (and all other Non-Android devices) ?

No, this is only for non-ubuntu distros where Libertine doesn't make sense and doesn't work.

**Review by Flohack74 — DISMISSED** [review]

**Comment by Flohack74** [comment]
LGTM @mariogrip

File: CMakeLists.txt [hunk 1]
```diff
@@ -55,6 +54,13 @@ pkg_check_modules(UBUNTU_DOWNLOAD_MANAGER REQUIRED ubuntu-download-manager-clien
 pkg_check_modules(NOTIFY REQUIRED libnotify)
 pkg_check_modules(APPARMOR REQUIRED libapparmor)
 
+pkg_check_modules(LIBERTINE libertine)
+if(LIBERTINE_FOUND)
+   message("Bulding with libertine support")
```
Comment (dobey) [inline CMakeLists.txt]: Typo. _Bulding_ instead of _Building_.

**Review by dobey — CHANGES_REQUESTED** [review]

**Review by dobey — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (dobey) [reply CMakeLists.txt]: Fixed this directly to move things along.

**Pull request merged**